bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -217,7 +328,7 @@ public synchronized int getClusterNodeCount(String clusterId, String zoneId) {
    * @return the {@link Cluster} if it was set. If the cluster is not found, throw a {@link
    *     NullPointerException}.
    */
-  public synchronized Cluster getCluster(String clusterId, String zoneId) {
+  public Cluster getCluster(String clusterId, String zoneId) {
     Cluster response = null;
     for (Cluster cluster : getClusters().getClustersList()) {
```

Comment:
Do we allow a single project in a single zone to have duplicate clusterids?  Last time I asked, I thought the answer was no.  This is not allowed:

> Instance1:cluster1 (where cluster is in zone1)
> instance2:cluster1 (where cluster is in zone1)

This is:

> Instance1:cluster1 (where cluster is in zone1)
> instance2:cluster1 (where cluster is in zone2)